support-frontend/conf/PROD.public.conf

```diff
@@ -6,9 +6,21 @@
 
 stage="PROD"
 
-identity.webapp.url="https://profile.theguardian.com"
-identity.api.url="https://idapi.theguardian.com"
-identity.production.keys=true
+identity {
+  webapp.url="https://profile.theguardian.com"
+  api.url="https://idapi.theguardian.com"
+  production.keys=true
+
+  oauth {
+    client.id="0oa79m3ecvbJXDSqg417"
+    issuer.url="https://profile.theguardian.com/oauth2/aus3xgj525jYQRowl417"
+    audience="https://profile.theguardian.com/"
+    authorize.url="https://profile.theguardian.com/oauth2/aus3xgj525jYQRowl417/v1/authorize"
+    token.url="https://profile.theguardian.com/oauth2/aus3xgj525jYQRowl417/v1/token"
+    callback.url="https://support.theguardian.com/oauth/callback"
```

Comment:
How come the oath config wasn’t in this file already (like it was for CODE/DEV)? Has it not been used in production?

Reply:
The Code/Dev config was introduced in #4997 and I should have added the Prod config then.  But it hasn't been used in Prod yet.

Reply:
Perfect, that makes sense